gliderpy/fetchers.py

```diff
@@ -13,34 +13,41 @@
 
 from erddapy import ERDDAP
 
-
 OptionalStr = Optional[str]
 
 # This is hardcoded to the IOOS glider DAC.
 # We aim to support more sources in the near future.
 _server = "https://gliders.ioos.us/erddap"
 
-ifremer_vars = [
-    "time",
-    "latitude",
-    "longitude",
-    "PSAL",
-    "TEMP",
-    "PRES",
-    "platform_deployment",
-]
-
-ooi_vars = [
-    "latitude",
-    "longitude",
-    "ctdgv_m_glider_instrument_practical_salinity",
-    "ctdgv_m_glider_instrument_sci_water_temp",
-    "ctdgv_m_glider_instrument_sci_water_pressure_dbar",
-    "time",
-    "quality_flag",
-    "trajectory"
-]
-
+server_vars = {
+    "https://gliders.ioos.us/erddap": [
+        "depth",
+        "latitude",
+        "longitude",
+        "salinity",
+        "temperature",
+        "time",
+    ],
+    "http://www.ifremer.fr/erddap": [
+        "time",
+        "latitude",
+        "longitude",
+        "PSAL",
+        "TEMP",
+        "PRES",
+        "platform_deployment",
+    ],
+    "https://erddap-uncabled.oceanobservatories.org/uncabled/erddap": [
```

Comment:
Using the URL as the key is cumbersome. I see you already opened #38. Do you want to implement that here or in a new PR? I'm fine merging this as-is to reduce the scope of the next PR BTW.

Reply:
This PR is already getting pretty big. I suggest we merge it now and I'll implement friendly server names in a new PR